### ISSUE TITLE
Feature request: diff between patch(set)s

### ISSUE THREAD

**Issue by aarongable** [esprehn/chromium-codereview#56]
In the normal rietveld UI, you can diff against previous patchsets. This feature is particularly useful for multiple rounds of review (what have they changed since I last left comments?) and for relands, where you can re-upload the original CL as the first patchset and then your fix as the second patchset.

That doesn't appear to be possible in this UI.

**Comment by esprehn** [comment]
I don't think this feature is useful since it often has tons of rebase stuff in it. I don't think we should add it until there's more people asking.

**Comment by aarongable** [comment]
Ok, I'll get the whole infra team down here :) Seriously, we all use this feature every day.

The killer argument is: it is the only way to simultaneously see their responses to your comments, and the code changes they made along with those responses.

**Comment by esprehn** [comment]
I'll add this once Rietveld has a decent API for it. I don't want to do more screen scraping.

**Comment by jyasskin** [comment]
I really want this for https://codereview.chromium.org/812853002/. I don't want to re-review the whole patch, just the changes since my review.

It may be possible to identify points where the author sync'ed and let people diff against that baseline (like the internal google code review tool does), but I'd be happy with just what Rietveld's old UI has.
